frontend/src/components/filters/descriptions.js

```diff
@@ -114,6 +114,20 @@ export const SAMPLE_FILTERS = {
     width: 250,
     detached: true,
   },
+  qPCR_status: {
+    type: FILTER_TYPE.SELECT,
+    key: "qPCR_status",
+    label: "qPCR Selection Status",
+    placeholder: "All",
+    mode: "multiple",
+    options: [
+      { label: "Positive", value: "Positive" },
```

Comment:
Move these status to the constants.js file. Make sure they have a name that distinguish them from Project status.